demo/install-demo.sh

```diff
@@ -20,6 +20,7 @@ DEMO=$1
 
 SCRIPTS_DIR=$BASEDIR/test/e2e/scripts
 
+export AERAKI_TAG=1.2.3
```

Comment:
```
if [ -z "$AERAKI_TAG" ]; then
  export AERAKI_TAG="1.2.3"
fi
```
I think it is a better way, right?

Reply:
I agree. User can specify the Aeraki version by setting this environment variable.